tests/prov/test_prov_pods_deployment.py

```diff
@@ -334,3 +334,71 @@ def test_40374(self):
         self.collect_sb = False
         self.destroy_flag = True
         self.log.info("===Test Completed===")
+
+    @pytest.mark.lc
+    @pytest.mark.cluster_deployment
+    @pytest.mark.tags("TEST-41907")
+    def test_41907(self):
+        """
+        Test to verify the multidata pod being deployed
+        """
+        config_list = self.deploy_obj.get_durability_config(num_nodes=len(self.worker_node_list))
```

Comment:
This method calculates sns, dix values considering 1, 2 and 3 CVG. 
So this might not always work with giving cvg as 1 in test_deployment method.

Reply:
we are not using the value of cvg retrieved from this method

Reply:
Or you can pick the first config returned which will be using cvg count as 1.